<ui version="4.0" >
 <class>UIAddAbbreviation</class>
 <widget class="QDialog" name="UIAddAbbreviation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Code Template</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>3</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="2" column="1" >
    <widget class="QComboBox" name="cbLanguages" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lLanguage" >
     <property name="text" >
      <string>Language :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lDescription" >
     <property name="text" >
      <string>Description :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leDescription" />
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leTemplate" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lTemplate" >
     <property name="text" >
      <string>Template :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UIAddAbbreviation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIAddAbbreviation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
